INeedCoffee.Tests/UserTests.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Xunit;
+using INeedCoffee;
+
+namespace INeedCoffee.Tests
+{
+    public class UserTests
+    {
+        [Fact]
+        public void CaptureUserCoordinates_ShouldWork()
+        {
+            UserCoordinates userCoordinates = new UserCoordinates();
+
+            string[] coordinates = { "45.7838552", "24.1515281" };
+
+            UserCoordinatesCapture.Capture(userCoordinates, coordinates);
+
+            string actual = userCoordinates.Latitude.ToString() + " " + userCoordinates.Longitude.ToString();
+
+            string expected = "45.7838552 24.1515281";
+
+            Assert.Equal(expected, actual);
```

Comment:
When I built the test case, I based it on the from the UserCoordinatesCapture file, but definitely should’ve checked individual properties instead of being based on that method as I’ve done in UserCoordinatesValidator